test/cppsuite/test_harness/test.h

```diff
@@ -106,14 +106,28 @@ class test : public database_operation {
     run()
     {
         int64_t cache_size_mb, duration_seconds;
-        bool enable_logging;
-
+        bool enable_logging, statistics_logging;
+        configuration *statistics_config;
+        std::string statistics_type;
         /* Build the database creation config string. */
         std::string db_create_config = CONNECTION_CREATE;
 
-        /* Get the cache size, and turn logging on or off. */
+        /* Get the cache size. */
         cache_size_mb = _config->get_int(CACHE_SIZE_MB);
-        db_create_config += ",statistics=(fast),cache_size=" + std::to_string(cache_size_mb) + "MB";
+
+        /* Get the statistics configuration for this run. */
+        statistics_config = _config->get_subconfig(STATISTICS_CONFIG);
+        statistics_type = statistics_config->get_string(TYPE);
+        statistics_logging = statistics_config->get_bool(ENABLE_LOGGING);
+
+        /* Don't forget to delete. */
+        delete statistics_config;
+
+        db_create_config += ",statistics=(" + statistics_type + "),";
+        db_create_config += statistics_logging ? std::string(STATISTICS_LOG) : "";
+        db_create_config += ",cache_size=" + std::to_string(cache_size_mb) + "MB";
```

Comment:
if statistics_logging is false, you will end up with two successive ","

Reply:
6ea12d410